src/content/docs/en/guides/backend/sentry.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Monitor your Astro Site with Sentry
+description: How to monitor your Astro site with Sentry
+type: backend
+service: Sentry
+stub: false
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+[Sentry](https://sentry.io) offers a comprehensive application monitoring and error tracking service designed to help developers identify, diagnose, and resolve issues in real-time.
+
+Read more on our blog about [Astro's partnership with Sentry](https://astro.build/blog/sentry-official-monitoring-partner/) and Sentry's Spotlight dev toolbar app that brings a rich debug overlay into your Astro development environment. Spotlight shows errors, traces, and important context right in your browser during local development.
+
+Sentry's Astro SDK enables automatic reporting of errors and tracing data in your Astro application.
+
+## Project Configuration
+
+A full list of prerequisites can be found in [the Sentry guide for Astro](https://docs.sentry.io/platforms/javascript/guides/astro/#prerequisites).
+
+## Install
+
+Sentry captures data by using an SDK within your application’s runtime.
+
+Install the SDK by running the following command for the package manager of your choice in the Astro CLI:
+
+```bash
+npx astro add @sentry/astro
+```
+The astro CLI installs the SDK package and adds the Sentry integration to your `astro.config.mjs` file.
+
+## Configure
+
+Add your DSN and project name and token to your `astro.config.mjs`:
```

Comment:
I think we can expand a little more on sdk keys/credentials. Added a little note for new projects since the sentry setup guide is provide all the necessary information.

```suggestion
To configure the Sentry integration, you need to provide the following credentials in your `astro.config.mjs` file.

1. **Client key (DSN)** - You can find the DSN in your Sentry project settings under *Client keys (DSN)*.
2. **Project name** - You can find the project name in your Sentry project settings under *General settings*.
3. **Auth token** - You can create an auth token in your Sentry organization settings under *Auth tokens*.

:::note
If you are creating a new Sentry project, select Astro as your platform to get all the necessary information to configure the SDK.
:::
```